hw/ip/pwm/doc/dv/index.md

```diff
@@ -78,29 +78,34 @@ All test sequences reside in `hw/ip/pwm/dv/env/seq_lib`.
 The `pwm_base_vseq` virtual sequence is extended from `cip_base_vseq` and serves as a starting point.
 All test sequences are extended from `pwm_base_vseq`.
 It provides commonly used handles, variables, functions and tasks that the test sequences can simple use / call.
+
 Some of the most commonly used tasks / functions are as follows:
-* task 1:
-* task 2:
+* initialize_pwm: wait for out of reset then program `REGEN` register
+* program_pwm_cfg_regs: program the `CFG.CLK_DIV` and `CFG.DC_RESN` for all channels
+* program_pwm_mode_regs: program the operation modes (Standard/Blinking/Heartbeat) for all channels 
+* start_pwm_channels: program `CFG.CNTR_EN`, `PWM_EN`, `INVERT` registers for all channels
+* run_pwm_channels: wait for a certain number of pulses generated in activate channels then stop all channels
 
 #### Functional coverage
 To ensure high quality constrained random stimulus, it is necessary to develop a functional coverage model.
 The following covergroups have been developed to prove that the test intent has been adequately met:
-* cg1:
-* cg2:
+* TBD:
```

Comment:
nit: always use TODO, otherwise, we need to grep many key words like TBD, FIXME etc

Reply:
@weicaiyang Change to TODO